service/worker/batcher/workflow.go

```diff
@@ -367,27 +367,27 @@ func startTaskProcessor(
 				err = processTask(ctx, limiter, task, batchParams, client,
 					batchParams.CancelParams.CancelChildren,
 					func(workflowID, runID string) error {
-						return client.RequestCancelWorkflowExecution(ctx, &shared.RequestCancelWorkflowExecutionRequest{
+						return client.RequestCancelWorkflowExecution(ctx, &types.RequestCancelWorkflowExecutionRequest{
```

Comment:
I am not entirely sure if we need to convert the error. Seems like EntityNotExist is the only error we care about and it's already updated? Same for terminate and signal?

Reply:
We don't need that here. Both batcher and failovermanager workers were simple enough to update in one shot. Thus no thrift related types (including errors) left there.
This particular error returned here is checked against `*types.EntityNotExistsError` just as you mentioned, which is already converted as well.